book_source/03_topical_pages/07_remote_access/01_pecan_api.Rmd

```diff
@@ -43,14 +43,26 @@ The currently implemented functionalities include:
   * [`GET /api/pfts/`](#get-apipfts): Search for PFT(s) using search pattern based on PFT name, PFT type & Model type
   * [`GET /api/pfts/{pft_id}`](#get-apipftspft_id): Fetch the details of specific PFT
 
+* __Formats:__
+  * [`GET /api/formats/`](#get-apiformats): Search for format(s) using search pattern based on format name & mime type
+  * [`GET /api/formats/{format_id}`](#get-formatsformat_id): Fetch the details of specific Format
+  
+* __Inputs:__
+  * [`GET /api/inputs/`](#get-apiinputs): Search for inputs needed for a PEcAn workflow based on `model_id` & `site_id`
+  * [`GET /api/inputs/{input_id}`](#get-apiinputsinput_id) *: Download the desired input file
+  
 * __Workflows:__
   * [`GET /api/workflows/`](#get-apiworkflows): Retrieve a list of PEcAn workflows
   * [`POST /api/workflows/`](#post-apiworkflows): Submit a new PEcAn workflow
   * [`GET /api/workflows/{id}`](#get-apiworkflowsid): Obtain the details of a particular PEcAn workflow by supplying its ID
+  * [`GET /api/workflows/{id}/status`](#get-apiworkflowsidstatus): Obtain the status of a particular PEcAn workflow
+  * [`GET /api/workflows/{id}/file/{filename}`](#get-apiworkflowsidfilefilename): Download the desired file from a PEcAn workflow
```

Comment:
Yes, `/api/workflows/{id}` returns the list of files so that the user knows which to request. As of now, it fetches files in the workflow directory only....coz the ones in the `workflow/runs/` are fetched by the `/api/runs/{run_id}/input/{filename}` & those in the `workflow/out/` are fetched by the `/api/runs/{run_id}/input/{filename}`

Reply:
Don't want to hold up this PR further, but we should also find a way to fetch files in the PFT directory, as a lot of sensitivity analysis results get squirreled away in there. Likewise, the PDA and SDA analyses often create additional subdirectories when run.